### PR TITLE
Lobby

### LINKED ISSUE #13: Room before game
We need a room before the game start, where players joins, choose their pseudo and wait for the "host" to start the game.

### PR DESCRIPTION
**Pull request by vch9**
The lobby:
* list of players
* url to the game
* button to start the game
lobby

It is really ugly, but it's enough to start a real game

Closes #13

### CODE REVIEW

File: public/css/lobby.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#body {
+    min-height: 100%;
+    min-width: 100%;
+}
+
+ul{
+    overflow: hidden;
```
Comment (maiste) [inline public/css/lobby.css]: Isn't it `overflow-x` ?

File: public/scripts/render_lobby.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ * Render the pre-game lobby.
+ */
+
+
+function createTitle(text, to) {
+    let title = document.createElement("div");
+    title.id = "title";
+    title.innerHTML = text;
+    document.getElementById(to).appendChild(title);
+
```
Comment (maiste) [inline public/scripts/render_lobby.js]: Trailing new line

File: public/scripts/render_lobby.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+}
```
Comment (maiste) [inline public/scripts/render_lobby.js:11]: Missing newline

File: public/scripts/render_lobby.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+function renderStart(players, link) {
+    document.getElementById("startGame").innerHTML = "";
+    let start = document.createElement("button");
+    start.id = "start";
+    start.innerHTML = "Start the game";
+
+    document.getElementById("startGame").appendChild(start);
+
+    document.querySelector('#game').addEventListener("click", function (){
+        // TODO: Start the game
```
Comment (maiste) [inline public/scripts/render_lobby.js]: Can you add a `console.log` to avoid a null problem ?

**Review by maiste — CHANGES_REQUESTED** [review]

File: public/css/lobby.css (hunk 1 above)
Comment (vch9) [reply public/css/lobby.css]: Actually, both should be hidden. The css isn't meant to display a LOT of players.

File: public/scripts/render_lobby.js (hunk 4 above)
Comment (vch9) [reply public/scripts/render_lobby.js]: log when the button is pressed ?

Comment (maiste) [reply public/scripts/render_lobby.js]: Just replace the //Todo by `console.log("Game'). Indeed actually it generates an error in the logs.

**Review by maiste — APPROVED** [review]
Ok for merging.

**Pull request merged**